src/content/docs/guides/run-a-taiko-node.mdx

```diff
@@ -73,6 +73,7 @@ After getting the address of the Holesky node, set the following L1 node endpoin
 
 - `L1_ENDPOINT_HTTP=http://192.168.1.15:8545`
 - `L1_ENDPOINT_WS=ws://192.168.1.15:8546`
+- `L1_BEACON_HTTP=ws://192.168.1.15:5052`
```

Comment:
shouldn't this be http?

Reply:
my bad, yes, I copied the ws and I forgot to change to `http`